helpers/measures.ts

```diff
@@ -173,14 +173,20 @@ function floatFormatter(value: string | number): string {
   return numberFormatter(value, 1, 5);
 }
 
-function percentFormatter(value: string | number, options: { decimals?: number } = {}): string {
+function percentFormatter(
+  value: string | number,
+  { decimals, singleTrailingZero }: { decimals?: number; singleTrailingZero?: boolean } = {}
```

Comment:
NP: How about `omitExtraDecimalZeros`?

Reply:
Wouldn't that be misleading? What is "extra"? For example, in 85.000, are the "extra" 0s all 3, or just the last 2? In our case, we want 85.000 to show as 85.0, not 85.

Reply:
Ah, but then again, if we have 84.5900, and set `decimals` to 3 and `singleTrailingZero` to true, it will show 84.59, not 84.590, which might be expected from this flag 🤔 . But I'm not sure we'll ever run into such a case 😅 .

Reply:
Whatever the name is, maybe a comment explaining exactly what will happen with this flag would make sense.

Reply:
Exactly the point I was about to answer, and the case is much much more common than a integer. In fact on next, only a handful of projects have a integer % (excluding 100 and 0).